plugins-builtin/src/test/scala/za/co/absa/enceladus/plugins/buildin/kafka/ControlInfoSerSuite.scala

```diff
@@ -16,11 +16,11 @@
 package za.co.absa.enceladus.plugins.buildin.kafka
 
 import org.apache.commons.io.IOUtils
-import org.scalatest.FunSuite
+import org.scalatest.funsuite.AnyFunSuite
```

Comment:
Can I ask why these changes?

Reply:
This is due to scala test version update.

https://www.scalatest.org/release_notes/3.1.0#scalatest_deprecations